Patches/CheckGameEndPatch.cs

```diff
@@ -557,231 +524,43 @@ public bool CheckGameEndByLivingPlayers(out GameOverReason reason)
 
             else if (totalNKAlive == 0) // total number of nks alive 0
             {
-                if (Crew == 0 && Imp == 0) // crew and Imp both 0 everyone dead
+                if (roleCounts[CountTypes.Crew] == 0 && roleCounts[CountTypes.Impostor] == 0) // Crew and Imp both 0, everyone is dead
```

Comment:
Since `roleCounts[CountTypes.Impostor]` and `roleCounts[CountTypes.Crew]` are frequently used in the code, it would be helpful to create temporary variables for them, similar to `totalNKAlive`. These variables can be named `impCount` and `crewCount`, and they can be placed on line 523 - 524 of the code.

Reply:
I mean remember this data and use it in code so as not to have to access the Dictionary very often